Source/player.cpp

```diff
@@ -83,17 +84,16 @@ void WalkSides(int pnum, Point vel, DirectionSettings const &walkParams)
 {
 	auto &player = plr[pnum];
 
-	int const x = walkParams.map.x + player.position.tile.x;
-	int const y = walkParams.map.y + player.position.tile.y;
+	Point const nextPosition = walkParams.map + player.position.tile;

```

Comment:
Don't think we have const before the type any where else.
```suggestion
	const Point nextPosition = walkParams.map + player.position.tile;
```

Reply:
My bad, I'm an `east const` user. I'll change them to `west cont`s, but we might wanna consider `east const` for[ its benefits](https://isocpp.org/wiki/faq/const-correctness#const-ref-alt).

Reply:
I think I've sorted all of the consts out.